<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="6" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="test.test.rrrTest" duration-ms="1805024" started-at="2013-10-17T07:42:12Z" finished-at="2013-10-17T08:12:17Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="1805024" started-at="2013-10-17T07:42:12Z" finished-at="2013-10-17T08:12:17Z">
      <class name="test.test.rrrTest">
        <test-method status="FAIL" signature="main()[pri:0, instance:test.test.rrrTest@36a0829c]" name="main" duration-ms="1805008" started-at="2013-10-17T10:42:12Z" finished-at="2013-10-17T11:12:17Z">
          <exception class="com.thoughtworks.selenium.SeleniumException">
            <message>
              <![CDATA[ERROR: Command timed out]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.thoughtworks.selenium.SeleniumException: ERROR: Command timed out
	at com.thoughtworks.selenium.HttpCommandProcessor.throwAssertionFailureExceptionOrError(HttpCommandProcessor.java:112)
	at com.thoughtworks.selenium.HttpCommandProcessor.doCommand(HttpCommandProcessor.java:106)
	at com.thoughtworks.selenium.DefaultSelenium.open(DefaultSelenium.java:369)
	at test.test.rrrTest.testSelenium(rrrTest.java:60)
	at test.test.rrrTest.main(rrrTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- com.thoughtworks.selenium.SeleniumException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main -->
        <test-method status="PASS" signature="main4()[pri:0, instance:test.test.rrrTest@36a0829c]" name="main4" duration-ms="16" started-at="2013-10-17T11:12:17Z" finished-at="2013-10-17T11:12:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main4 -->
        <test-method status="PASS" signature="main6()[pri:0, instance:test.test.rrrTest@36a0829c]" name="main6" duration-ms="0" started-at="2013-10-17T11:12:17Z" finished-at="2013-10-17T11:12:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main6 -->
        <test-method status="PASS" signature="main7()[pri:0, instance:test.test.rrrTest@36a0829c]" name="main7" duration-ms="0" started-at="2013-10-17T11:12:17Z" finished-at="2013-10-17T11:12:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main7 -->
        <test-method status="PASS" signature="main8()[pri:0, instance:test.test.rrrTest@36a0829c]" name="main8" duration-ms="0" started-at="2013-10-17T11:12:17Z" finished-at="2013-10-17T11:12:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main8 -->
        <test-method status="PASS" signature="main5()[pri:0, instance:test.test.rrrTest@36a0829c]" name="main5" duration-ms="0" started-at="2013-10-17T11:12:17Z" finished-at="2013-10-17T11:12:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main5 -->
      </class> <!-- test.test.rrrTest -->
    </test> <!-- Command line test -->
  </suite> <!-- test.test.rrrTest -->
</testng-results>
